infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/CorfuReplicationClusterManagerBaseAdapter.java

```diff
@@ -45,7 +46,8 @@ public void prepareToBecomeStandby() {
         corfuReplicationDiscoveryService.prepareToBecomeStandby();
     }
 
-    public int queryReplicationStatus() {
-        return corfuReplicationDiscoveryService.queryReplicationStatus();
+    public LogReplicationMetadata.ReplicationStatusVal queryReplicationStatus() {
+        return corfuReplicationDiscoveryService.queryReplicationStatus(
+            topologyConfig.getClusters(0).getId());
```

Comment:
It's not guaranteed to always place the local cluster in the 0th index position right?
Am I mistaken?